.github/workflows/release_ffi.yaml

```
@@ -58,43 +59,35 @@ jobs:
       cluster_name: ${{ env.cluster_name }}
       build_ffi: ${{ steps.tasks.outputs.build_ffi }}
 
-
   build_fbs_header:
     if: ${{ needs.init.outputs.build_ffi == 'true' }}
     needs: init
-    runs-on: self-hosted
+    runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: swatinem/rust-cache@v2
-        with:
-          prefix-key: "v0-rust"
-          cache-on-failure: "false"
-          cache-targets: "false"
-          cache-directories: |
-            ~/.cache/cachepot/
-            ~/.local
-          key: bjv3
-        timeout-minutes: 5
-        continue-on-error: true
-      - uses: ./.github/actions/rust-cache
-        with:
-          rust-toolchain: rustfmt, clippy
-      - uses: ./.github/actions/cache
-        with:
-          key: ${{ runner.os }}-build
       - name: Authenticate AWS CLI
         uses: aws-actions/configure-aws-credentials@v1
         with:
           role-to-assume: arn:aws:iam::578672650824:role/github-actions-knox-app
           aws-region: us-east-2
+      - run: |
+          sudo chown -R "$USER:$USER" "/home/$USER"
       - name: Install deps
         run: ./container/deps.sh --force grpc_health_probe flatc cargo-make protoc buf
       - run: |
           sudo chown -R "$USER:$USER" "/home/$USER"
-      - name: Generate FFI Header
-        run: |
-          source .env.cachepot.local
+
+      - run: which buf
+        
+      - run: cd ./registry_resolver && buf generate buf.build/knox-networks/registry-mgmt:f7ff6f57030c418e886459a18b35645e
+      - run: chmod -R 0777 ./registry_resolver/src/gen
+      
+      - run: |
+          cargo make ffi-build
```

```suggestion
      - run: cargo make ffi-build
```
A lot of these don't need a `|`